Sources/Apodini/Semantic Model Builder/SharedSemanticModelBuilder.swift

```diff
@@ -66,6 +66,7 @@ class SharedSemanticModelBuilder: SemanticModelBuilder, InterfaceExporterVisitor
         super.register(handler: handler, withContext: context)
         
         let operation = context.get(valueFor: OperationContextKey.self)
+        let serviceType = context.get(valueFor: ServiceTypeContextKey.self)
```

Comment:
see above, I myself also switched approach and did not include the `customDescription` set via `DescriptionModifier` in `Endpoint` but rather accessed the context in a [function](https://github.com/Apodini/Apodini/pull/117/files#diff-b0f79c59dcf157375189003b00adde1270c0c73de3ec8906ecf8d2cb850c83f9R56) of the `OpenAPIPathBuilder` object, where I needed it.

Reply:
maybe we should think about moving `serviceType`, `serviceName` and `methodName` (in `Endpoint+GRPC`) also out of `Endpoint` as this - from my understanding - is only supposed reflect "shared" properties of endpoints.

Reply:
As said above, I think `serviceType` is actually an addition that could be of use for other exporters as well.
I agree for `serviceName` and `methodName`, and moved them out of the `Endpoint` extension.